yt_dlp/extractor/pornhub.py

```diff
@@ -509,6 +559,9 @@ def extract_list(meta_key):
             'tags': extract_list('tags'),
             'categories': extract_list('categories'),
             'cast': extract_list('pornstars'),
+            'production': extract_list('production'),
+            'language_spoken': extract_list('langSpoken'),
+            'model_attributes': get_model_attributes(),
```

Comment:
Those are not valid info dict fields. https://github.com/yt-dlp/yt-dlp/blob/master/yt_dlp/extractor/common.py#L261

Reply:
Does this mean they cannot be added to the extractor?

Reply:
I don't think there are any nice ways to map this data to valid info dict fields. You could propose adding new fields, but due to being very site-specific, I don't think they would be accepted.

If you need this data, you should look in to creating a plugin https://github.com/yt-dlp/yt-dlp?tab=readme-ov-file#developing-plugins
You can even re-use your code changes submitted here.

Reply:
`language` could be put inside the formats. Would it be reasonable to merge attributes/production with tags/categories respectively? Or are they too different?